sycl/test-e2e/Graph/Explicit/depends_on.cpp

```diff
@@ -37,11 +37,11 @@ int main() {
 
   Queue.submit([&](handler &CGH) { CGH.ext_oneapi_graph(ExecGraph); }).wait();
 
-  constexpr int ref = 42 * 2;
+  constexpr int Ref = 42 * 2;
   std::vector<int> Output(N);
   Queue.memcpy(Output.data(), Arr, N * sizeof(int)).wait();
-  for (int i = 0; i < N; i++)
-    assert(Output[i] == ref);
+  for (size_t i = 0; i < N; i++)
+    assert(check_value(i, Output[i], Ref, "Ref"));
```

Comment:
I think the `Output[i]` and `Ref` parameters here should be swapped when looking at the definition of `check_value()` since the reference value should come first. The variable name is also incorrect in this case and should be `"Output"`.

Reply:
Good catch! Updated these cases